README.md

```diff
@@ -1,8 +1,35 @@
 # local-storage-operator
 Operator for local storage
 
+```mermaid
+graph LR
+    PVC -->|Requests local storage using a StorageClass| StorageClass
+    StorageClass -->|Defines a LocalVolume| LocalVolume
+    LocalVolume -->|Represents a physical storage device already mounted on a node| Node
+    Pod -->|References a PVC| PVC
+    PV -->|References| LocalVolume
+    StorageClass -->|Creates| PV
```

Comment:
No, LSO creates PVs. StorageClass is just for reference and is not really used anywhere in PV creation / deletion / binding.

Reply:
Thanks for catching that. Changed the flow so that LSO creates the PVs in https://github.com/openshift/local-storage-operator/commit/f8acb21ec74b51b9fdfde6ab9f8c0a8d71ad3ea9